libraries/AP_WayBack/test/test-wayback.ino

```diff
@@ -0,0 +1,182 @@
+/*
+*/
+
+#define SERIAL_USB 1
+
+#define LED_PIN PB1
+
+#include "Arduino.h"
+
+#define HAL_BOARD_SITL     3
+#define HAL_BOARD_SMACCM   4  // unused
+#define HAL_BOARD_PX4      5
+#define HAL_BOARD_LINUX    7
+#define HAL_BOARD_VRBRAIN  8
+#define HAL_BOARD_QURT     9
+#define HAL_BOARD_REVOMINI 10
+
+#define CONFIG_HAL_BOARD HAL_BOARD_SITL
+
+#include "AP_WayBack.h"
+#include "AP_Math.h"
+
+AP_WayBack track;
+
+
+#define SERIAL_BUFSIZE 128
+char buffer[SERIAL_BUFSIZE];
+
+
+void getSerialLine(char *cp ){      // получение строки
```

Comment:
please make comments in English (again, please use the rest of the codebase as an example)